--- a/src/curobo/util/sample_lib.py
+++ b/src/curobo/util/sample_lib.py
@@ -377,9 +377,7 @@
                 if self.sample_ratio[k] == 0.0:
                     continue
                 n_samples = round(sample_shape[0] * self.sample_ratio[k])
-                s_shape = torch.Size(
-                    [n_samples], device=self.tensor_args.device, dtype=self.tensor_args.dtype
-                )
+                s_shape = torch.Size([n_samples])
                 # if(k == 'halton' or k == 'random'):
                 samples = self.sample_fns[k](sample_shape=s_shape)
                 # else:
